setup.cfg

```diff
@@ -0,0 +1,11 @@
+[flake8]
+max-line-length=120
+exclude=.git,__pycache__
+
+[isort]
```

Comment:
The settings from mozregression are probably good to use (minus the parts that are specific to glean-dictionary, which seem correct). As mentioned above, define them in pyproject.toml:

https://github.com/mozilla/mozregression/blob/master/pyproject.toml#L12